DESCRIPTION

```diff
@@ -29,8 +29,10 @@ Authors@R: c(
     person("Sunni", "Raleigh", email = "sunnigraleigh@gmail.com", role = "ctb"),
     person("Anna", "Ballou", email = "afballou16@gmail.com", role = "ctb"),
     person("Jane", "Bang", email = "carolbjy@gmail.com", role = "ctb"),
+    person("Kara", "Van Allen", email = "kkv354@gmail.com", role = "ctb"),
     person("Jessica", "Keast", email = "jeskeast@gmail.com", role = "ctb"),
     person("Fatima", "Keita", email = "fkeita898@gmail.com", role = "ctb"))
+
```

Comment:
You'll need to remove this blank line. Having it causes the package build to yield an error.